src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java

```diff
@@ -36,4 +43,13 @@ public RoleList clickOnRoleMenu() {
         CommonEvents.clickButton(roleMenu);
         return new RoleList();
     }
+
+    public void clickOnAuditsMenu() {
+        CommonEvents.clickButton(auditsMenu);
+    }
+
+    public AuditList clickOnAuditMenu() {
+        CommonEvents.clickButton(auditMenu);
```

Comment:
before method should be included in this method, just to avoid to create new page object

Reply:
just for this time I will accept this type.